src/components/screens/ExperimentCreator.vue

```diff
@@ -0,0 +1,1348 @@
+                          <Textarea v-model="abstractText" :id="$scopedId('input-abstractText')" rows="10" />
+                          <label :for="$scopedId('input-abstractText')">Abstract</label>
+                        </span>
+                      </TabPanel>
+                      <TabPanel header="Preview">
+                        <div class="mavedb-wizard-rendered-markdown" v-html="markdownToHtml(abstractText)"></div>
+                      </TabPanel>
+                    </TabView>
+                    <span v-if="validationErrors.abstractText" class="mave-field-error">{{ validationErrors.abstractText
+                    }}</span>
+                  </div>
+                </div>
+                <div class="mavedb-wizard-row">
+                  <div class="mavedb-wizard-help">
+                    <label>
+                      A condensed description of the data analysis, starting from raw sequence data,
+                      suitable for a specialist audience of MAVE researchers.
+                    </label>
+                    <div class="mavedb-help-small">
+                      May be formatted using <a target="_blank" href="https://daringfireball.net/projects/markdown/syntax">Markdown</a>.
+                      Should include:
+                      <ul>
+                        <li>variant library construction methods,</li>
+                        <li>description of the functional assay, including model system and selection type,</li>
+                        <li>sequencing strategy and sequencing technology, and</li>
+                        <li>structure of biological or technical replicates (if applicable).</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <div class="mavedb-wizard-content field">
+                    <TabView>
+                      <TabPanel header="Edit">
+                        <span class="p-float-label">
+                          <Textarea v-model="methodText" :id="$scopedId('input-methodText')" rows="10" />
+                          <label :for="$scopedId('input-methodText')">Methods</label>
+                        </span>
+                      </TabPanel>
+                      <TabPanel header="Preview">
+                        <div class="mavedb-wizard-rendered-markdown" v-html="markdownToHtml(methodText)"></div>
+                      </TabPanel>
+                    </TabView>
+                    <span v-if="validationErrors.methodText" class="mave-field-error">{{ validationErrors.methodText }}</span>
+                  </div>
+                </div>
+                <div class="mavedb-wizard-row">
+                  <div class="mavedb-wizard-help">
+                    <label>
+                      Digital Object Identifier
+                    </label>
+                    <div class="mavedb-help-small">
+                      Preprints or publications that are not indexed by PubMed can be included via the DOI field.
```

Comment:
Let's make the help text for DOIs the same on this page as already exists on the score set page.

Reply:
I don't see the DOI in `ScoreSetCreator` page, but it's in `ScoreSetEditor` page. Do we miss it in `ScoreSetCreator` page?  
@afrubin Does creating a new score set have DOI? I remember we had it before.

Reply:
Yeah it got dropped out somewhere, not sure why. I added it back in the target gene wizardification change with this help text:

```
                    <div class="mavedb-wizard-help">
                      <label :id="$scopedId('input-doiIdentifiers')">
                        The DOIs of any digital resources associated with the score set.
                      </label>
                      <div class="mavedb-help-small">
                        Please note: If you would like to associate publications with this score set via their DOI, please do not do so here.
                        Instead, use the publication identifiers field below.
                      </div>
                    </div>
```